packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -843,17 +847,24 @@ function emitBasicOperation(
     namespaceHierarchies.push(operationGroupName);
   }
 
+  let hasApiVersionInOperation = false;
```

Comment:
can we join this and hasApiVersionInClient in a single variable like apiVersionLocation: "client" | "operation"?

Reply:
I have moved hasApiVersionInClient into emitter options as we have agreed that we skip the consideration of modular and RLC are inconsistent for now and keep hasApiVersionInOperation as the local variable.